data/NV/incentives.json

```diff
@@ -458,21 +458,21 @@
       "pos_rebate"
     ],
     "item": "other",
-    "program": "nv_nVEnergy_residentialAirconditioningProgram",
+    "program": "nv_nVEnergy_residentialAirConditioning",
     "amount": {
       "type": "dollar_amount",
-      "number": 3600,
-      "minimum": 1700,
-      "maximum": 3600
+      "number": 3060,
+      "minimum": 1445,
+      "maximum": 3060
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "Up to $3,600 income qualified instant rebate when you replace your old air condition system with higher efficiency equipment.",
-      "es": "Un reembolso instantáneo de hasta $3,600 dólares que cumpla con el nivel de ingresos cuando sustituya su viejo sistema de aire acondicionado por un equipo de eficiencia superior. "
+      "en": "Up to $3,060 income qualified instant rebate when you replace your old air condition system with higher efficiency equipment.",
+      "es": "Un reembolso instantáneo de hasta $3,060 dólares que cumpla con el nivel de ingresos cuando sustituya su viejo sistema de aire acondicionado por un equipo de eficiencia superior. "
     },
-    "low_income": "nv-nv-energy"
+    "low_income": "nv-nv-energy-residential-air-conditioning"
   },
   {
     "id": "NV-24",
```

Comment:
Should this have an exclusion relationship with NV-21? It looks like the income-qualified version of the same thing. Same with NV-25 and NV-22